.processes/community.md

```diff
@@ -0,0 +1,12 @@
+# Community Processes
+
+The HOPR Association _tech_ members, _community_ members and [Project Owner](./development.md#legend) have agreed on the following interaction processes to streamline allocation of _tech_ resources.
+
+## Creating Issues
+
+Community team is expected to create github issues in [this repository](https://github.com/hoprnet/hopr-devrel) using [this template](https://github.com/hoprnet/hopr-devrel/blob/main/.github/ISSUE_TEMPLATE/request-for-tech-work.md).
```

Comment:
For specific recurring community-tech task, such as "minting NFT", please use [this RFM template](https://github.com/hoprnet/hopr-devrel/blob/main/.github/ISSUE_TEMPLATE/request-for-minting-hoprboost-nft.md)

Reply:
updated